src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -178,6 +178,41 @@ public enum NuGetLogCode
         /// </summary>
         NU1213 = 1213,
 
+        /// <summary>
+        /// Uncategorized FatalProtocolException
+        /// </summary>
+        NU1300 = 1300,
+
+        /// <summary>
+        /// Feed was Unauthorized
+        /// </summary>
+        NU1301 = 1301,
+
+        /// <summary>
+        /// Fallback folder or local feed folder does not exist
+        /// </summary>
+        NU1302 = 1302,
+
+        /// <summary>
+        /// Feed was Forbidden
+        /// </summary>
+        NU1303 = 1303,
+
+        /// <summary>
+        /// Url was Not Found, server exists
+        /// </summary>
+        NU1304 = 1304,
```

Comment:
I like how this matches up with 404, kinda.

Reply:
that was the thought for 401, 403, 404, and 407.